global_config.js

```diff
@@ -0,0 +1,46 @@
+// global_config.js
+
+var Parse = require('parse/node').Parse,
+    PromiseRouter = require('./PromiseRouter');
+
+var router = new PromiseRouter();
+
+function updateGlobalConfig(req) {
+  if (!req.auth.isMaster) {
+    return Promise.resolve({
+      status: 401,
+      response: {error: 'unauthorized'},
+    });
+  }
+
+  return req.config.database.rawCollection('_GlobalConfig')
+    .then(coll => coll.findOneAndUpdate({ _id: 1 }, { $set: req.body }, { returnOriginal: false }))
+    .then(response => {
+      return { response: { params: response.value.params } }
+    })
+    .catch(() => ({
+      status: 404,
+      response: {
+        code: 103,
```

Comment:
I've updated with https://github.com/theill/parse-server/commit/bd4546f444232e2fdf014b4809753142f964c4c5 which should use correct error message.

Should we consider an explicit error message for a missing `params` attribute in payload? As I see the current production response from a `GET /config` is to return an object with a `params` attribute. So we really don't want users to `POST /config` without that sub attribute because they wouldn't be able to read it out again anyway, since we explicitly return only the objects `params` attribute in our `GET` method.